students/jemielity_kamil/lesson_01_basics/apple_sharing.py

```diff
@@ -0,0 +1,10 @@
+import math
+
+N = int(input("Number of students: "))
```

Comment:
Capital letters should be used only for constants e.g.
MY_NAME = "Kamil"

Please use lower case instead:
n = int(input("Number of students: "))